### PR TITLE
Release v4.0.0-beta.2

### PR DESCRIPTION
**Pull request by jakolehm**
## Changes since v4.0.0-beta.1

- Default PageRegistration.routePath to path based on id (#1374)
-  Remove Makefile from all intree extensions (#1312)
- Simplify pages/menus/registry extension api internal implementation (#1364)
- Exit pipeline if extensions build or tests fail (#1370)
- Docs: fixing typos in styling (#1363)
- Fix Landing Page width (#1359)
- Upgrade cache pipeline task (#1366)
- Update styling docs (#1339)

### CODE REVIEW

File: static/RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Here you can find description of changes we've built into each release. While we try our best to make each upgrade automatic and as smooth as possible, there may be some cases where you might need to do something to ensure the application works smoothly. So please read through the release highlights!
 
-## 4.0.0-beta.1 (current version)
+## 4.0.0-beta.2 (current version)
```
Comment (Nokel81) [inline static/RELEASE_NOTES.md:5]: Shouldn't the changes be add to this?

Comment (jakolehm) [reply static/RELEASE_NOTES.md:5]: There are no user visible (major) changes? Release page will have full list of changes.

Comment (Nokel81) [reply static/RELEASE_NOTES.md:5]: Ah okay, I guess we count "Fix Landing Page width (#1359)" as a bugfix then?

**Comment by Nokel81** [comment]
Rebase with master now that #1312 has been merged should fix CI

**Review by Nokel81 — APPROVED** [review]

**Pull request merged**